### PR TITLE
Add open source license

### PR DESCRIPTION
**Pull request by alvrogd**
To welcome contributions from everybody! :)

- Apache v2.0 `LICENSE`
- Bundled third-party software `NOTICE`

### CODE REVIEW

**Review by ruifm — APPROVED** [review]

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+or under their own terms.
+
+We clearly identify all third-party software included within this product by
+either:
+
+1. Placing it in a separate directory structure, with a `LICENSE` or
+   `LICENSE.txt` file at its root, detailing the applicable licensing
+   conditions.
+
+2. Including the applicable licensing conditions at the top of each file.
```
Comment (daniel-otero) [inline NOTICE:12]: It seems easier to read to me.

```suggestion
   1. Placing it in a separate directory structure, with a `LICENSE` or
      `LICENSE.txt` file at its root, detailing the applicable licensing
      conditions.

   2. Including the applicable licensing conditions at the top of each file.
```

**Review by daniel-otero — APPROVED** [review]
Great! :partying_face: :partying_face: :partying_face:

File: NOTICE (hunk 1 above)
Comment (alvrogd) [reply NOTICE]: Agreed! Done :)

**Review by jgonzac — APPROVED** [review]

**Review by inaki-amatria — APPROVED** [review]
LGTM. Don't forget to squash the "fixup!" before mering :-)

**Pull request merged**